packages/uicore/src/components/MultiTypeInput/MultiTypeInput.tsx

```diff
@@ -38,7 +38,7 @@ export interface ExpressionAndRuntimeTypeProps<T = unknown> extends Omit<LayoutP
   value?: AcceptableValue
   multitypeInputValue?: MultiTypeInputType
   defaultValueToReset?: AcceptableValue
-  width?: CSSProperties['width']
+  width?: number
```

Comment:
This is reverting a recently merged change, could you rebase again?

Reply:
Done. Thanks for catching it. I updated the branch from github and it caused this